en/tutorials-and-examples/cms/authorization.rst

```diff
@@ -88,6 +88,12 @@ queries. Lets use bake to generate a basic policy:
 .. code-block:: bash
 
     bin/cake bake policy --type entity Article
+    
+    and
+    
+    bin/cake bake policy --type table Articles   (from cakephp v2 book)
```

Comment:
Why did you need this policy? All the checks in the tutorial are against the entity.

Reply:
I don't need it, but without it the cake would generate an error. Looking at the cake v2 documentation, I saw this command line and ran it, so the cake worked, although there were other errors.

Reply:
> All the checks in the tutorial are against the entity.

I did not understand this sentence. Could you clarify it?

Reply:
> I did not understand this sentence. Could you clarify it?

All of the tutorial authorization checks are made against an entity and not the table. Perhaps you were trying to use the application between generating the policy with bake and updating it with the correct methods?